docs/pages/bssw/bssw_content_highlighting.md

```diff
@@ -40,7 +40,7 @@ Some points to note:
 ````
 - Each side of a slide can hold (1) link to a content page or (2) an image. For example: For a blog post, left side can hold link to the blog post and right side can hold a deck image.
 - The `link` is a relative url of the article hosted on the BSSw.io site. For ex: The [article](https://bssw.io/blog_posts/give-thanks) with url `https://bssw.io/blog_posts/give-thanks` would be referred to `blog_posts/give-thanks` in the link.
-- The `image` is also the relative url to the [BSSw.io images repo](https://github.com/betterscientificsoftware/images) with a strict path. For ex: If we need to put [this image](https://github.com/betterscientificsoftware/images/blob/master/Blog_1119_seasonal.png) in the carousel, you would specify it as `images/raw/master/Blog_1119_seasonal.png`.
+- The `image` is a strange url somewhat relative to the BSSw.io repo as well except it has a `/raw/` subdirectory in it. For ex: If we need to put [this image](https://github.com/betterscientificsoftware/bssw.io/blob/master/images/Blog_1119_seasonal.png) in the carousel, then we would specify it as `images/raw/master/Blog_1119_seasonal.png`.  (NOTE:  The `raw` and `master` parts make no sense in this context since it comes from the same branch as the `bssw.io` site files itself but this is a holdover from when there was a separate `images` github repo and has not yet been updated after the move of the images to the main `bssw.io` github repo.  It is expected this will be fixed at some point in the site generator for bssw.io and therefore fixed in this file.) 
```

Comment:
I suggest we hold on changing this file.  Parallactic has been revising how this is handled to be (more) consistent with what we do elsewhere.

Reply:
> I suggest we hold on changing this file. Parallactic has been revising how this is handled to be (more) consistent with what we do elsewhere.

Okay, I will revert the change to this file and we can leave this as a todo.  Since this file is not linked to from the main menu, no-one will accidentally find it.  (I was only able to view it by manually typing in the html file name on the locally generated Jekyll site.)